xl-v1-multipart-common.go

```diff
@@ -175,7 +182,11 @@ func (xl xlObjects) writeUploadJSON(bucket, object, uploadID string, initiated t
 		wg.Wait()
 		return errXLWriteQuorum
 	}
-	return nil
+	return reduceErrs(errs, []error{
```

Comment:
> this will help readability
> 
> ignoreErrs := []error{errDiskNotFound, errFaultyDisk, errDiskAccessDenied}
> return reduceErrs(errs, ignoreErrs)

agreed!